sdk/protocol-plugins/src/plugins/morphoblue/implementation/MorphoProtocolPlugin.ts

```diff
@@ -0,0 +1,421 @@
+        value: Number(marketInfo[5]),
+      }),
+    }
+  }
+
+  /**
+   * @name _getMarketParams
+   * @description Get the market parameters from the market ID
+   * @param morphoLendingPoolId The lending pool ID
+   * @returns The market parameters
+   */
+  private async _getMarketParams(
+    morphoLendingPoolId: IMorphoLendingPoolId,
+  ): Promise<MorphoMarketParameters> {
+    const morphoBlueAddress = await this._getContractAddress({
+      chainInfo: morphoLendingPoolId.protocol.chainInfo,
+      contractName: MorphoProtocolPlugin.MorphoBlueContractName,
+    })
+    const marketParamsId = morphoLendingPoolId.marketId
+
+    const [marketParameters] = await this.ctx.provider.multicall({
+      contracts: [
+        {
+          abi: morphoBlueAbi,
+          address: morphoBlueAddress.value,
+          functionName: 'idToMarketParams',
+          args: [marketParamsId],
+        },
+      ],
+      allowFailure: false,
+    })
+
+    const debtToken = await this.ctx.tokensManager.getTokenByAddress({
+      address: Address.createFromEthereum({ value: marketParameters[0] }),
+      chainInfo: morphoLendingPoolId.protocol.chainInfo,
+    })
+
+    if (!debtToken) {
+      throw new Error(
+        `Invalid debt token address: ${marketParameters[0]} for chain ${morphoLendingPoolId.protocol.chainInfo.name}`,
+      )
+    }
+
+    const collateralToken = await this.ctx.tokensManager.getTokenByAddress({
+      address: Address.createFromEthereum({ value: marketParameters[1] }),
+      chainInfo: morphoLendingPoolId.protocol.chainInfo,
+    })
+
+    if (!collateralToken) {
+      throw new Error(
+        `Invalid collateral token address: ${marketParameters[1]} for chain ${morphoLendingPoolId.protocol.chainInfo.name}`,
+      )
+    }
+
+    const lltv = new BigNumber(String(marketParameters[4]))
+      .div(new BigNumber(10).pow(MorphoLLTVPrecision))
+      .multipliedBy(100)
+      .toNumber()
+
+    return {
+      debtToken,
+      collateralToken,
+      oracle: Address.createFromEthereum({ value: marketParameters[2] }),
+      irm: Address.createFromEthereum({ value: marketParameters[3] }),
+      lltv: Percentage.createFrom({ value: lltv }),
+    }
+  }
+
+  /**
+   * @name _getLiquidationThreshold
+   * @description Get the liquidation threshold for the given morpho lending pool ID
+   * @param marketInfo The market info
+   * @param marketCollateralPriceInDebt The market collateral price in debt
+   * @returns The liquidation threshold
+   */
+  private _getLiquidationThreshold(
+    marketInfo: MorphoMarketInfo,
+    marketCollateralPriceInDebt: IPrice,
+  ): IRiskRatio {
+    const riskRatioValue = marketInfo.totalBorrowAssets
+      .toBN()
+      .div(marketCollateralPriceInDebt.toBN().multipliedBy(marketInfo.totalSupplyAssets.toBN()))
+      .multipliedBy(100)
+
+    return RiskRatio.createFrom({
+      ratio: Percentage.createFrom({
+        value: riskRatioValue.toNumber(),
+      }),
+      type: RiskRatioType.LTV,
+    })
```

Comment:
As far as I remember liquidation threshold for a pool in morpho is just `lltv` that you get from chain. The thing that you calculated is `LTV` of a pool.

Reply:
You are right. This should just be the Pool LLTV. I'll change it